src/ray/gcs/gcs_server/job_info_handler_impl.h

```diff
@@ -36,6 +38,7 @@ class DefaultJobInfoHandler : public rpc::JobInfoHandler {
 
  private:
   gcs::RedisGcsClient &gcs_client_;
+  const std::shared_ptr<gcs::GcsPubSub> &gcs_pub_sub_;
```

Comment:
It is better to use `std::shared_ptr<gcs::GcsPubSub> gcs_pub_sub_` directly.

Reply:
fixed